content/en/docs/faq/_index.md

```diff
@@ -75,3 +75,17 @@ cert-manager currently has some [limited experimental support] for this resource
 [`kubectl certificates` command]: https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#certificate
 [Request signing process]: https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#request-signing-process
 [limited experimental support]: ../usage/kube-csr/
+
+
+### Why do certificates issued by a Vault Issuer sometimes not have the intermediate CA in `tls.crt`?
+
+cert-manager `v1.4` introduced more sophisticated certificate chain parsing to sort the chain and determine the root certificate, see the [release notes for `v1.4`] 
+cert-manager now parses all the certificates (leaf and CAs) returned by Vault to build an ordered chain. The root-most certificate from the chain will be written to `ca.crt` and the rest of the ordered chain will be written to `tls.crt` of the `Secret` resource for that certificate.
```

Comment:
nit: is it two sentences separated with a new line? There might be a missing `.` I think
```suggestion
cert-manager `v1.4` introduced more sophisticated certificate chain parsing to sort the chain and determine the root certificate, see the [release notes for `v1.4`].
cert-manager now parses all the certificates (leaf and CAs) returned by Vault to build an ordered chain. The root-most certificate from the chain will be written to `ca.crt` and the rest of the ordered chain will be written to `tls.crt` of the `Secret` resource for that certificate.
```